src/js/SignInUI.js

```diff
@@ -23,10 +25,15 @@ export default class SignIn {
       AuthAPI.login({
         identifier: user.value.trim(),
         password: pass.value.trim(),
-      }).catch((e) => {
-        const errorMsg = e.response.data.message[0].messages[0].message;
-        SignIn.errorLog(errorMsg, passError);
-      });
+      })
+        .then(()=> {
+          loginModal.classList.remove('active');
+          new UserUI().init(true);
```

Comment:
why we need to pass `true` to `init()` method?

Reply:
bug)